app/scripts/modules/core/src/region/regionSelectField.directive.js

```diff
@@ -1,13 +1,72 @@
 'use strict';
 
 const angular = require('angular');
+import { react2angular } from 'react2angular';
+
+import { RegionSelectField } from './RegionSelectField';
 
 module.exports = angular
   .module('spinnaker.core.region.regionSelectField.directive', [])
+  .component(
+    'reactRegionSelectField',
+    react2angular(RegionSelectField, [
+      'regions',
+      'component',
+      'field',
+      'account',
+      'provider',
+      'onChange',
+      'labelColumns',
+      'fieldColumns',
+      'readOnly',
+    ]),
+  )
+  .component('regionSelectFieldProxy', {
```

Comment:
seems like this intermediate component might not be necessary?  or is it?

Have a look at this similar example in https://github.com/spinnaker/deck/blob/0e984db58d5d41ca67f7b8ca98d6972977a021d2/app/scripts/modules/core/src/entityTag/notifications/entityNotifications.component.ts

Reply:
You're right, it's not necessary. 👍